.storybook/preview-head.html

```diff
@@ -0,0 +1,22 @@
+<script>
+  window.global = window;
+</script>
+<style>
+    @import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap');
+
+    :root {
+        --app-prefer-width: 1024px;
+        --app-extended-width: 1500px;
+
+        --border-radius: 16px;
+        --font: 'Lato', sans-serif;
+    }
+
+    body, main  {
+        margin: 0;
+        padding: 0;
+        min-height: 100vh;
+        font-family: var(--font);
+        background-color: silver;
+    }
+</style>
```

Comment:
are there any way to import already existing index.css?

Reply:
good idea, fixed